netconf.c

```diff
@@ -1523,9 +1524,10 @@ get_process_action (struct netconf_session *session, xmlNode *rpc, xmlNode *node
                     VERBOSE ("XPATH: malformed query\n");
                     *ret = send_rpc_error_full (session, rpc, NC_ERR_TAG_MALFORMED_MSG, NC_ERR_TYPE_RPC,
                                                 "XPATH: malformed query", NULL, NULL, true);
-                    cleanup_on_xpath_error (session, attr, split, ns_href, ns_prefix);
+                    cleanup_on_xpath_error (session, attr, split, ns_href, ns_prefix, path);
                     return -1;
                 }
+                g_free (path);
```

Comment:
This was indeed a new leak, g_strstrip modifies the string in place and the cleanup of that string would be when the vector was freed, so no need to free path originally.